lib/jobs.js

```diff
@@ -61,8 +61,21 @@ function start(buildConfig, callback) {
  * @param {Function}   callback                   Callback fn(error, result)
  */
 function stop(buildConfig, callback) {
-    return redis.hdel(`${queuePrefix}buildConfigs`, buildConfig.buildId)
-        .then(() => executor.stop(buildConfig))
+    const stopConfig = { buildId: buildConfig.buildId };
+
+    return redis.hget(`${queuePrefix}buildConfigs`, buildConfig.buildId)
```

Comment:
Could you make this keyname `const`? I think we don't like repeat such.

Reply:
Okay, I fixed them.